enterprise/backend/test/metabase_enterprise/advanced_permissions/common_test.clj

```diff
@@ -26,7 +26,7 @@
   [graph f]
   (let [all-users-group-id  (u/the-id (perms-group/all-users))
         current-graph       (get-in (perms/data-perms-graph) [:groups all-users-group-id])]
-    (premium-features-test/with-premium-features #{:advanced-permissions}
+    (premium-features-test/with-premium-features #{:advanced-permissions :audit-app}
```

Comment:
Can you change this to use `with-additional-premium-features`?

Then I think this macro can only set `:advanced-permissions`, and `:audit-app` can be enabled in just the test that requires it.

Reply:
Done